### ISSUE TITLE
Nonsense data returned by scan

### ISSUE THREAD

**Issue by kimble4** [Anrijs/Aranet4-ESP32#3]
Using the example `Scanner.ino` (library cloned from GitHub, as the 1.2.1 release doesn't seem to include this functionality), and an Aranet4 Home with firmware version 1.4.14, I'm getting nonsense values:

```
15:20:35.636 -> Scanning BT devices...
15:20:40.644 -> Found 38 devices
15:20:40.644 -> -----------------------------
15:20:40.644 -> Aranet4 25583    v1.4.14
15:20:40.644 ->     CO2:          44804 ppm
15:20:40.644 ->     Temperature:  1049.65 C
15:20:40.644 ->     Pressure:     1181.5 C
15:20:40.644 ->     Humidity:     611 %
15:20:40.644 ->     Battery:      60 %
15:20:40.644 ->     Interval:     38 s
15:20:40.677 ->     Ago:          77 s
15:20:40.677 -> -----------------------------
```

Meanwhile, using `BasicRead.ino` (fixing the typo on line 44), I get valid readings:

```
15:19:02.435 -> Connecting...
15:19:03.562 -> Aranet4 read OK
15:19:03.562 -> CO2:          1160 ppm
15:19:03.562 -> Temperature:  21.55 C
15:19:03.562 -> Pressure:     1006.5 C
15:19:03.562 -> Humidity:     46 %
15:19:03.562 -> Battery:      99 %
15:19:03.562 -> Interval:     60 s
15:19:03.562 -> Ago:          1 s
15:19:03.562 -> Waiting 59 seconds for next measurement
```

Any ideas?  Presumably the data format isn't quite right.

I'd quite like to use the BLE scan if possible, as it will integrate well with existing code to read other sensors...

**Comment by Anrijs** [comment]
Should be fixed in v1.4.1

**Comment by kimble4** [comment]
Yep, that's fixed it.  Thanks.
